src/stcal/jump/jump.py

```diff
@@ -14,7 +14,11 @@
 def detect_jumps(frames_per_group, data, gdq, pdq, err,
                  gain_2d, readnoise_2d, rejection_thresh,
                  three_grp_thresh, four_grp_thresh, max_cores, max_jump_to_flag_neighbors,
-                 min_jump_to_flag_neighbors, flag_4_neighbors, dqflags):
+                 min_jump_to_flag_neighbors, flag_4_neighbors, dqflags,
+                 after_jump_flag_dn1=0.0,
```

Comment:
As these are keywords, I thought that defaults might be nice.  Fine with removing them, but just wanted to confirm that this is policy.

Reply:
I now realize that if we do not have defaults here, then all the tests need updating to explicitly have all 4 keywords with these defaults.  Having keywords with defaults means that existing tests did not need to be modified.  This was the motivation for using keywords and not regular parameters. 

So, removing defaults means adding more to every test.  If removing the defaults is still desired, say so and I'll start cutting and pasting.  But I would leave the defaults here if possible to avoid this.

Reply:
In that case I withdraw my comment. It certainly doesn't hurt to have them there. Just a matter of convention and/or consistency.

Reply:
Ok.  Will leave them as they are.